frontend/integration-tests/views/metalkube/dashboards.view.ts

```diff
@@ -0,0 +1,101 @@
+import { browser, element, by , $$ } from 'protractor';
+import { waitForNone } from '../../protractor.conf';
+
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+export const isLoaded = () => browser.wait(untilNoLoadersPresent).then(() => browser.sleep(2000));
```

Comment:
IMO sleep should not be part of this function, but used separately instead

Reply:
This was taken from crudView.isLoaded(). I think it's best to leave it this way.

Reply:
ok